index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src='https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous'></script>
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="justify-between">
+      <div id="logo-placeholder">
+        <img src="./images/logo.png" alt="logo">
+      </div>
+      <div id="humberger">
+        <div id="top-bar"></div>
```

Comment:
- [ ] I would kindly suggest you wrap the hamburger icon inside a `nav` tag.
According to the requirement,
The hamburger button is on the right, and has a `nav` tag.

The nav tag is for semantics reason, and the right position for convention and user experience for mobile devices.